connexta/eve
# Eve 14

#### What does this PR do?
Created Jenkins environment so that any changes to the github connexta/eve repo will trigger Jenkin/Jenkinsfile to rebuild the docker image and the project. 
Test deployment part has not been included yet...!

#### Who is reviewing it?

@vinamartin @Kjames5269 @bankent1 @alchucam @mcalcote

#### How should this be tested?
making changes to the github repo, and observe rebuild and Jenkins repository log in devops/eve-wallboard

#### Screenshots
current github setup for Jenkins
![image](https://user-images.githubusercontent.com/40583917/60292542-39b18480-98d2-11e9-8b3a-141634b44dac.png)


#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
